src/ngraph/runtime/aligned_buffer.hpp

```diff
@@ -32,7 +34,9 @@ namespace ngraph
 class ngraph::runtime::AlignedBuffer
 {
 public:
-    AlignedBuffer(size_t byte_size, size_t alignment);
+    AlignedBuffer(size_t byte_size,
+                  size_t alignment,
+                  std::shared_ptr<ngraph::runtime::Allocator> allocator = nullptr);
```

Comment:
make the default value for allocator `ngraph_alloc` and remove all checks for m_allocator == nullptr